packages/wdio-browserstack-service/src/service.ts

```diff
@@ -84,16 +83,18 @@ export default class BrowserstackService implements Services.ServiceInstance {
     afterTest(test: Frameworks.Test, context: never, results: Frameworks.TestResult) {
         const { error, passed } = results
 
-        this._fullTitle = (
-            /**
-             * Jasmine
-             */
-            test.fullName ||
-            /**
-             * Mocha
-             */
-            `${test.parent} - ${test.title}`
-        )
+        // Jasmine
+        if (test.fullName) {
+            const testSuiteName = test.fullName.slice(0, test.fullName.indexOf(test.description || '') - 1)
```

Comment:
What happens if `indexOf` returns `-1`, can we be sure that this will never happen?

Reply:
I copied this from the sauce-service. I am sure `-1` cannot happen since the test description (`it`) is always a substring of the fullname (`describe` + `it`).